### ISSUE TITLE
[Bug] batch_norm computational problem

### ISSUE THREAD

**Issue by chengven027-intellif** [apache/tvm#12708]
hi，tvm
When I load a onnx model. I found the tvm result is different ONNX-RT result. The reason is because of `batch_norm` op. But I can`t found any error with the op.  Can  you  help me to analyze it. Thanks very much.
### Expected behavior
The model result between tvm and onnx-RT is equal.

### Actual behavior
The model result between tvm and onnx-RT is not equal.

### Environment
x86

### Steps to reproduce
run the model with tvm and onnx-RT.

**Comment by chengven027-intellif** [comment]
The  model is in :
https://github.com/chengven027-intellif/model/blob/master/convert_remove.onnx
@AndrewZhaoLuo  Can you help me? thanks.
I check the file : src/relay/transforms/simplify_inference.cc  `BatchNormToInferUnpack`. Looks like it have no error.  And now, I have many model test result error with the `batch_norm` op .

**Comment by AndrewZhaoLuo** [comment]
Hey, thanks for tracking this down. I'll take a look Friday.

**Comment by chengven027-intellif** [comment]
Hey, @AndrewZhaoLuo  Have you seen this PR, and any update with me? thanks.

**Comment by AndrewZhaoLuo** [comment]
Take a look in the weekend

**Comment by chengven027-intellif** [comment]
It has been solved. Thanks.
